src/Engine/EngineBase.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Engine;
+
+use App\Exception\OcrException;
+
+abstract class EngineBase
+{
+
+    /**
+     * @param string $imageUrl
+     * @param string $lang
```

Comment:
`string|null` which I thought the linter would catch?

Reply:
Done.

I guess it's a MediaWiki codesniffer rule that catches that? But Phan finds it correctly now.